src/main/java/uk/gov/hmcts/cmc/claimstore/events/claim/ClaimIssuedEvent.java

```diff
@@ -39,8 +39,8 @@ public Claim getClaim() {
         return Optional.ofNullable(pin);
     }
 
-    public String getSubmitterName() {
-        return submitterName;
+    public Optional<String> getSubmitterName() {
```

Comment:
This is just wrong. You had to make it optional because the way `uk.gov.hmcts.cmc.claimstore.services.DocumentManagementSealedClaimDocumentService#uploadClaim` is implemented. To summarise bad decision led to other bad decision. Please revert and reimplement mentioned method in a better way.

Reply:
actioned